src/applications/vaos/new-appointment/components/ConfirmationPage/index.jsx

```diff
@@ -59,7 +53,7 @@ export class ConfirmationPage extends React.Component {
       data.facilityType !== FACILITY_TYPES.COMMUNITY_CARE
     ) {
       // Remove parse function when converting this call to FHIR service
-      this.props.fetchFacilityDetails(parseFakeFHIRId(data.vaFacility));
```

Comment:
This `vaFacility` is already being parsed downstream in `getLocation()`.  When we call this action already parsed, it gets stored in the `facility.details.983` key, but our selector `getChosenFacilityDetails()` is looking for `facility.details.var983`